misc/de-metas-common/de-metas-common-rest_api/src/main/java/de/metas/common/rest_api/payment/JsonInboundPaymentInfo.java

```diff
@@ -47,6 +47,12 @@
 @JsonDeserialize(builder = JsonInboundPaymentInfo.JsonInboundPaymentInfoBuilder.class)
 public class JsonInboundPaymentInfo
 {
+	@ApiModelProperty(required = true, //
+			dataType = "java.lang.String", //
+			value = "An identifier for the transaction being processed.")
+	@Nullable
+	String paymentId;
```

Comment:
```suggestion
			value = "An external identifier for the payment being posted to metasfresh. Translates to `C_Payment.ExternalId`")
	@Nullable
	String externalPaymentId;
```

...to make sure noone thinks it's about `C_Payment.C_Payment_ID`..